recipes/sphinxjp.themecore/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "0.2.0" %}
+{% set sha256 = "55729fd3f70b0073da007829b54f533296a9787f03d68a45d747192fcae8c2ad" %}
+
+package:
+  name: sphinxjp.themecore
+  version: {{ version }}
+
+source:
+  fn: sphinxjp.themecore-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/s/sphinxjp.themecore/sphinxjp.themecore-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - docutils
+    - sphinx
+
+  run:
+    - python
+    - docutils
+    - sphinx
+
+test:
+  imports:
+    - sphinxjp
+    - sphinxjp.themecore
+
+about:
+  home: http://bitbucket.org/shimizukawa/sphinxjp.themecore
+  license: MIT License
```

Comment:
Appears upstream does not include a license file. Please note that in a comment here and note that it should be added once there is one (see the [example]( https://github.com/conda-forge/staged-recipes/blob/3b962af8c4da15da4cb8386763538d6065e6c5fc/recipes/example/meta.yaml#L46-L49 )).

Also please raise an issue with upstream asking for a license file to be added to the repo and `MANIFEST.in`. Please make sure to clarify that the MIT license requires us to include it in all distributions. Hence why they really need to provide one so we can comply with their licensing terms. Please cross reference this PR in that issue and that issue in this PR once created.